docs/howto/edit-in-protege.md

```diff
@@ -0,0 +1,94 @@
+# Editing a term in protege
+
+Before you start:
+
+-   make sure you are working on a branch 
+-   make sure you have the editor's file open in Protege (in ODK ontologies, located in: ./src/ontology/ONT-edit.owl)
```

Comment:
can we be explicit that ONT = your ontology (oftentimes I don't understand things like this!)